framework/docs/source/how-to-migrate-from-openfl.rst

```diff
@@ -0,0 +1,537 @@
+are launched by the clients using the `flwr run` command. Beyond the code that is defined, Flower has the ability
+to insert dynamic changes through a configuration file, called ``pyproject.toml``. This can include application specific
+changes like hyperparameters, but also other information like ServerApp address, etc. Importantly, this file is shared
+between parties operating the ``ClientApp`` and ``ServerApp``. This concept directly maps to the Federated Learning
+Plan (FLPlan) concept in OpenFL captured in the ``plan.yaml`` file of every workspace.  
+
+.. code-block:: shell
+
+    # Flower pyproject.toml 
+    
+    ...
+
+    [tool.flwr.app.config]
+    num-server-rounds = 3
+    fraction-evaluate = 0.5
+    local-epochs = 1
+    learning-rate = 0.1
+    batch-size = 32
+
+    ...
+
+Server-side Code
+~~~~~~~~~~~~~~~~~~~~~
+
+In OpenFL, all of the aggregator-side code is configured via the `plan.yaml` file through the
+specification of different arguments. In Flower, the exact tasks performed by the server are more
+configurable through code. For example, aggregation algorithms are added through a ``Strategy``,
+and the logic to save models is added explictly. Here is a ``ServerApp`` 
+(akin to an OpenFL Aggregator) compatible with the prior code snippets:
+
+.. code-block:: python
+
+    import torch
+    from flwr.app import ArrayRecord, ConfigRecord, Context, MetricRecord
+    from flwr.serverapp import Grid, ServerApp
+    from flwr.serverapp.strategy import FedAvg
+    
+    from pytorchexample.task import Net, load_centralized_dataset, test
+    
+    # Create ServerApp
+    app = ServerApp()
+    
+    
+    @app.main()
+    def main(grid: Grid, context: Context) -> None:
+        """Main entry point for the ServerApp."""
+    
+        # Read run config
+        fraction_evaluate: float = context.run_config["fraction-evaluate"]
+        num_rounds: int = context.run_config["num-server-rounds"]
+        lr: float = context.run_config["learning-rate"]
+    
+        # Load global model
+        global_model = Net()
+        arrays = ArrayRecord(global_model.state_dict())
+    
+        # Initialize FedAvg strategy
+        strategy = FedAvg(fraction_evaluate=fraction_evaluate)
+    
+        # Start strategy, run FedAvg for `num_rounds`
+        result = strategy.start(
+            grid=grid,
+            initial_arrays=arrays,
+            train_config=ConfigRecord({"lr": lr}),
+            num_rounds=num_rounds,
+            evaluate_fn=global_evaluate,
+        )
+    
+        # Save final model to disk
+        print("\nSaving final model to disk...")
+        state_dict = result.arrays.to_torch_state_dict()
+        torch.save(state_dict, "final_model.pt")
+
+You'll notice that this each ``ServerApp`` should have specific logic for working
+with a given deep learning framework (in this case PyTorch) due to the saving of a final model.
```

Comment:
should be mention that this is entirely optional (it just happens to be that this template has a `ServerApp` that saves the pytorch model to disk) ?